cpp/modules/deck.gl/core/src/shaderlib/project/viewport-uniforms.cpp

```diff
@@ -172,24 +172,24 @@ auto calculateViewportUniforms(const std::shared_ptr<Viewport>& viewport, double
   // TODO(ilija@unfolded.ai): wrapLongitude and modelMatrix not set, set them to default values explicitly if they
   // aren't relevant?
   ViewportUniforms uniforms = {// Projection mode values
+                               .viewProjectionMatrix = Matrix4<float>{matrixAndOffset.viewProjectionMatrix},
```

Comment:
Just curious, what kind of order does gcc try to enforce? This doesn't seem to match order within the actual struct

Reply:
GCC was complaining that the initializers weren't in the order the fields were declared in.

Reply:
The order they are currently in on this branch does not match what's on `master`, that's why it looked pretty random to me. Perhaps we need to rebase to make sure it works after we merge